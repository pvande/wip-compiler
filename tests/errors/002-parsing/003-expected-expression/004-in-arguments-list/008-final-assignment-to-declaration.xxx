x := (a : int = 1, b : int = ) => () { }

// @TODO This should yield the same error.
// x := (a : int = 1, b : int =) => () { }
