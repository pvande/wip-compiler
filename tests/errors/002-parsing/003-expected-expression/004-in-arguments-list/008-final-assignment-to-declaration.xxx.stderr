Error: "Expected an expression"
In [1;37mtests/errors/002-parsing/003-expected-expression/004-in-arguments-list/008-final-assignment-to-declaration.xxx[0m on line [1;37m1[0m

> [0;36mx := (a : int = 1, b : int = ) => () { }[0m
  [0;31m                             ^          [0m
